### PR TITLE
Adjusts Chromatic settings

### PR DESCRIPTION
**Pull request by dzucconi**
Trying to see if I can get the spurious diff reviews to go away.

### CODE REVIEW

File: packages/palette/src/elements/BarChart/BarChart.story.tsx [hunk 1]
```diff
@@ -342,5 +342,5 @@ storiesOf("Components/BarChart", module)
         </Box>
       )
     },
-    { chromatic: { delay: 500, diffThreshold: 0.2 } }
```
Comment (dzucconi) [inline packages/palette/src/elements/BarChart/BarChart.story.tsx:345]: @damassi why were all the charts made to be more sensitive?

File: packages/palette/src/elements/Modal/Modal.story.tsx [hunk 2]
```diff
@@ -4,43 +4,63 @@ import { Button } from "../Button"
...
+    "Wide modal",
+    () => {
+      return (
+        <Modal show title="Modal Title" isWide onClose={() => null}>
+          Some example content
+        </Modal>
+      )
+    },
+    { chromatic: { delay: 500 } }
+  )
```
Comment (dzucconi) [inline packages/palette/src/elements/Modal/Modal.story.tsx:66]: OK well these look good now and just have to be accepted this one time

**Comment by dzucconi** [comment]
Ah OK I kept bumping the delay on the bar chart but they rest consistently at `left: 29.7031px` now in the diiff. Will lower those.

File: packages/palette/src/elements/BarChart/BarChart.story.tsx (hunk 1 above)
Comment (damassi) [reply packages/palette/src/elements/BarChart/BarChart.story.tsx:345]: Lol are they? Not sure 🤦

Comment (dzucconi) [reply packages/palette/src/elements/BarChart/BarChart.story.tsx:345]: 😆 https://www.chromatic.com/docs/threshold

> Chromatic’s default threshold is .063 which balances high visual accuracy with low false positives (for example, from artifacts like anti-aliasing). 0 is the most accurate. 1 is the least accurate.

I think all of the places where these were failing weren't due to the diffThreshold but rather views animating in. (So like the bar chart rising, the modal moving into the frame, etc)

**Comment by artsyit** [comment]
:rocket: PR was released in [`v9.8.1`](https://github.com/artsy/palette/releases/tag/v9.8.1) :rocket:

**Pull request merged**